Library/Homebrew/extend/pathname.rb

```diff
@@ -29,7 +29,7 @@ def compute_disk_usage
     if self.directory?
       @file_count = 0
       @disk_usage = 0
-      self.find do |f|
+      find do |f|
```

Comment:
Any reason not to alter other instances with the same `self.<method>` pattern, e.g., `self.directory?` above and `self.exist?` further down?